lib/getNextJobs.js

```diff
@@ -21,17 +21,17 @@ const getNextJobs = (workflowGraph, config) => {
     }
 
     workflowGraph.edges.forEach((edge) => {
-        // Check if edge src is specific branch commit with regexp
-        const edgeSrcBranchRegExp = new RegExp('^~commit:/(.+)/$');
+        // Check if edge src is specific branch commit or pr with regexp
+        const edgeSrcBranchRegExp = new RegExp('^~(pr|commit):/(.+)/$');
         const edgeSrcBranch = edge.src.match(edgeSrcBranchRegExp);
 
         if (edgeSrcBranch) {
-            // Check if trigger is specific branch commit
-            const triggerBranchRegExp = new RegExp('^~commit:(.+)$');
+            // Check if trigger is specific branch commit or pr
+            const triggerBranchRegExp = new RegExp('^~(pr|commit):(.+)$');
             const triggerBranch = config.trigger.match(triggerBranchRegExp);
 
-            if (triggerBranch) {
-                if (triggerBranch[1].match(edgeSrcBranch[1])) {
+            if (triggerBranch && triggerBranch[1] === edgeSrcBranch[1]) {
```

Comment:
These looks magic number so I think we should add some comments.

Reply:
Thanks, I added some comments.